plugins/inputs/clickhouse/clickhouse.go

```diff
@@ -205,6 +205,10 @@ func (ch *ClickHouse) Gather(acc telegraf.Accumulator) (err error) {
 	return nil
 }
 
+func (ch *ClickHouse) Stop() {
+	ch.client.CloseIdleConnections()
```

Comment:
This looks unrelated to the rest of the PR.

Reply:
I removed the Go 1.11 support code and this is moved over from `clickhouse_go1.12.go`.  I could split thee pull request?

Reply:
Sorry, I missed the deleted clickhouse files.  It makes sense that this was moved from clichouse_go1.12.go.  No need to split the pr.